src/main/java/com/perficient/movies/controllers/ActorController.java

```diff
@@ -1,11 +1,15 @@
 package com.perficient.movies.controllers;
 
 import com.perficient.movies.model.entities.Actor;
+import com.perficient.movies.model.mappers.ActorMapper;
 import com.perficient.movies.services.ActorService;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.*;
```

Comment:
Please, never import all the packages - never use "import some.libraries.*" is not a good practice.

Reply:
That makes it heavier, isn't it?. I changed it.